lib/external-secret.js

```diff
@@ -76,6 +76,8 @@ async function startWatcher ({
         clearTimeout(timeout)
       })
 
+      restartTimeout();
```

Comment:
Picky lint rules here
```suggestion
      restartTimeout()
```